modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -126,28 +126,82 @@ listen my-cluster-machine-config-api-22623
 
 listen my-cluster-apps-443
-        bind 192.168.1.100:443
-        mode tcp
-        balance roundrobin
-    option httpchk
-    http-check connect
-    http-check send meth GET uri /healthz/ready
-    http-check expect status 200
-        server my-cluster-worker-0 192.168.1.111:443 check port 1936 inter 10s rise 2 fall 2
-        server my-cluster-worker-1 192.168.1.112:443 check port 1936 inter 10s rise 2 fall 2
-        server my-cluster-worker-2 192.168.1.113:443 check port 1936 inter 10s rise 2 fall 2
+    bind 192.168.1.100:443
+    mode tcp
+    balance roundrobin
+  option httpchk
+  http-check connect
+  http-check send meth GET uri /healthz/ready
+  http-check expect status 200
+    server my-cluster-worker-0 192.168.1.111:443 check port 1936 inter 10s rise 2 fall 2
+    server my-cluster-worker-1 192.168.1.112:443 check port 1936 inter 10s rise 2 fall 2
+    server my-cluster-worker-2 192.168.1.113:443 check port 1936 inter 10s rise 2 fall 2
+
+listen my-cluster-apps-80
+   bind 192.168.1.100:80
+   mode tcp
+   balance roundrobin
+  option httpchk
+  http-check connect
+  http-check send meth GET uri /healthz/ready
+  http-check expect status 200
+    server my-cluster-worker-0 192.168.1.111:80 check port 1936 inter 10s rise 2 fall 2
+    server my-cluster-worker-1 192.168.1.112:80 check port 1936 inter 10s rise 2 fall 2
+    server my-cluster-worker-2 192.168.1.113:80 check port 1936 inter 10s rise 2 fall 2
+# ...
+----
++
+.Example HAProxy configuration with multiple listed subnets
+[source,terminal,subs="quotes"]
+----
+# ...
+listen my-cluster-api-6443
+    bind 192.168.1.100:6443
+    mode tcp
+    balance roundrobin
+  option httpchk
+  http-check connect
+  http-check send meth GET uri /readyz
+  http-check expect status 200
+    server my-cluster-master-2 192.168.222.111:6443 check inter 10s rise 2 fall 2
+    server my-cluster-master-0 192.168.333.221:6443 check inter 10s rise 2 fall 2
+    server my-cluster-master-1 192.168.444.212:6443 check inter 10s rise 2 fall 2
+
+listen my-cluster-machine-config-api-22623
+    bind 192.168.1.100:22623
+    mode tcp
+    balance roundrobin
+  option httpchk
+  http-check connect
+  http-check send meth GET uri /healthz
+  http-check expect status 200
+    server my-cluster-master-2 192.168.222.111:22623 check inter 10s rise 2 fall 2
+    server my-cluster-master-0 192.168.333.221:22623 check inter 10s rise 2 fall 2
+    server my-cluster-master-1 192.168.444.212:22623 check inter 10s rise 2 fall 2
+
+listen my-cluster-apps-443
+    bind 192.168.1.100:443
+    mode tcp
+    balance roundrobin
+  option httpchk
+  http-check connect
+  http-check send meth GET uri /healthz/ready
+  http-check expect status 200
+    server my-cluster-master-2 192.168.222.113:443 check inter 10s rise 2 fall 2
+    server my-cluster-master-0 192.168.333.222:443 check inter 10s rise 2 fall 2
+    server my-cluster-master-1 192.168.444.213:443 check inter 10s rise 2 fall 2
```

Comment:
```suggestion
    server my-cluster-worker-0 192.168.333.222:443 check inter 10s rise 2 fall 2
    server my-cluster-worker-1 192.168.444.213:443 check inter 10s rise 2 fall 2
```

Reply:
Thanks. Updated.

Reply:
@dfitzmau  I saw above still not updated for 443